scene/gui/option_button.cpp

```diff
@@ -573,6 +589,7 @@ void OptionButton::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_fit_to_longest_item"), &OptionButton::is_fit_to_longest_item);
 	ClassDB::bind_method(D_METHOD("set_allow_reselect", "allow"), &OptionButton::set_allow_reselect);
 	ClassDB::bind_method(D_METHOD("get_allow_reselect"), &OptionButton::get_allow_reselect);
+	ClassDB::bind_method(D_METHOD("set_disable_shortcuts", "disabled"), &OptionButton::set_disable_shortcuts);
```

Comment:
Bound methods must be documented. Use `bin/<godot executable> --doctool` to regenerate XML docs.

Reply:
See the docs of `MenuBar` and `MenuButton`.

https://github.com/godotengine/godot/blob/237bd0a615df8a0e57bc3d299894abece7b43a0c/doc/classes/MenuBar.xml#L59-L65

https://github.com/godotengine/godot/blob/237bd0a615df8a0e57bc3d299894abece7b43a0c/doc/classes/MenuButton.xml#L20-L26